src/client/CMakeLists.txt

```diff
@@ -1,4 +1,20 @@
+if(USE_SOUND)
+	set(sound_SRCS
+		${CMAKE_CURRENT_SOURCE_DIR}/sound_openal.cpp
+		${CMAKE_CURRENT_SOURCE_DIR}/sound.cpp)
```

Comment:
`sound.cpp` must be included regardless on `USE_SOUND` to have the DummySoundManager available.

Reply:
exact, fixing that